tests/unit/test-class-wc-payments.php

```diff
@@ -16,14 +18,16 @@ class WC_Payments_Test extends WCPAY_UnitTestCase {
 
 	public function set_up() {
 		// Mock the main class's cache service.
-		$this->_cache     = WC_Payments::get_database_cache();
-		$this->mock_cache = $this->createMock( WCPay\Database_Cache::class );
+		$this->_cache        = WC_Payments::get_database_cache();
+		$this->_card_gateway = WC_Payments::get_gateway();
+		$this->mock_cache    = $this->createMock( WCPay\Database_Cache::class );
```

Comment:
Feel like it would be better convention to define before initialising them, but not really a big deal: works fine either way.